pkg/chart/chart.go

```diff
@@ -373,3 +389,43 @@ func getRegistrySecretsPerDomain(appRepoSecrets []string, cluster, namespace, to
 	}
 	return secretsPerDomain, nil
 }
+
+// InitClient returns an HTTP client based on the chart details loading a
+// custom CA if provided (as a secret)
+// TODO(andresmgot): Using a custom CA cert is not supported by ORAS (neither helm), only using the insecure flag
```

Comment:
Do you know why (as in, is this intentional to require public tls certs)? What are the implications for private registries?